private/dev/Dashboard.js

```diff
@@ -125,7 +125,7 @@ export default () => {
       uppyDashboard.use(AwsS3Multipart, { companionUrl: COMPANION_URL, limit: 6 })
       break
     case 'xhr':
-      uppyDashboard.use(XHRUpload, { endpoint: XHR_ENDPOINT, limit: 6, bundle: true })
+      uppyDashboard.use(XHRUpload, { endpoint: XHR_ENDPOINT, limit: 6, bundle: false })
```

Comment:
Intentional? I don't mind, just making sure.

Reply:
Remote uploading doesn't work without this change. Probably better to have it like this in dev.

Reply:
You mean pause/resume doesn't? Uploading seems to be working fine.

Reply:
We it must be `false` to work:

https://github.com/transloadit/uppy/blob/7e4821885b3fccfdba5f2bc25aa20cf5ecb857ac/packages/%40uppy/xhr-upload/src/index.js#L641-L646

Reply:
Aaaah right.